bee-hive/cli/beeAI.py

```diff
@@ -32,12 +32,15 @@
 import os, sys, traceback
 
 from docopt import docopt
-from cli import *
-
-if __name__ == '__main__':
+from cli.cli import *
+  
+def runcommand():
```

Comment:
OK but we should be pythonic and call this:
```python
def run_cli():
  # ...
```

Reply:
updated as requested

Reply:
I still hate that we have to `import cli.cli ...` but I understand though we need to figure out a way to fix this. For me any time I see duplication or repetition is a place we can improve :)